specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-06-02-preview/managedClusters.json

```diff
@@ -6733,6 +6733,10 @@
           "$ref": "#/definitions/ManagedClusterSecurityProfileImageCleaner",
           "description": "Image Cleaner settings for the security profile."
         },
+        "imageIntegrity": {
+          "$ref": "#/definitions/ManagedClusterSecurityProfileImageIntegrity",
+          "description": "Image Integrity is a feature that works with Azure Policy to verify image integrity by signature. This will not have any effect unless policy is applied to enforce image signatures. See https://aka.ms/aks/image-integrity for how to use this feature via policy."
```

Comment:
keep consistent for all occurances of 'image integrity'?

Reply:
changed to all lower except first char of sentence